script.js

```diff
@@ -0,0 +1,62 @@
+let bookList = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author) {
+  bookList.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(bookList));
+}
+
+function removeBook(element, title, author) {
+  element.remove();
+  bookList = bookList.filter((book) => book.title !== title || book.author !== author);
+  localStorage.setItem('books', JSON.stringify(bookList));
+}
+
+const booksCollection = document.querySelector('.books');
+
+bookList.forEach((book) => {
+  const div = document.createElement('div');
+  div.innerHTML = `
+    <h2>${book.title}</h2>
+    <p>${book.author}</p>
+  `;
+
+  const removeBtn = document.createElement('button');
+  removeBtn.innerText = 'Remove';
+  removeBtn.addEventListener('click', () => {
+    removeBook(div, book.title, book.author);
+  });
+
+  div.appendChild(removeBtn);
+
+  booksCollection.appendChild(div);
+});
+
+const addBtn = document.querySelector('.add-book-btn');
+
+addBtn.addEventListener('click', () => {
+  // add book
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.